build.gradle

```diff
@@ -182,6 +182,7 @@ allprojects {
 
 dependencies{
 	compile project(":hasco")
```

Comment:
I am a bit confused about this dependency section. Why do we have these dependencies defined globally?

Reply:
it is not defined globally but for the super-project AILibs which consists of its sub-projects.